File: biz/biz-vk-sdk/src/main/java/ru/pet/kami/sdk/VkSdkApi.java
package ru.pet.kami.sdk;

import com.vk.api.sdk.objects.wall.responses.GetResponse;
import com.vk.api.sdk.queries.wall.WallGetFilter;

public interface VkSdkApi {
    GetResponse getPostsFromGroupWall(int ownerId, int count, int offset, WallGetFilter filter);
}

File: sdk-vk/src/main/java/ru/pet/kami/sdk/VkSdkApi.kt
package ru.pet.kami.sdk

import com.vk.api.sdk.objects.wall.responses.GetResponse
import com.vk.api.sdk.queries.wall.WallGetFilter

interface VkSdkApi {
    fun getPostsFromGroupWall(ownerId: Int, count: Int = 1000, offset: Int = 0): GetResponse?
}